src/main/java/org/grouplens/grapht/reflect/internal/ClassInstantiator.java

```diff
@@ -41,8 +41,10 @@
  * @author <a href="http://grouplens.org">GroupLens Research</a>
  */
 public class ClassInstantiator implements Instantiator {
-    private static final Logger logger = LoggerFactory.getLogger(ClassInstantiator.class);
-    
+
+    private static final Logger      logger        = LoggerFactory.getLogger(ClassInstantiator.class);
```

Comment:
This lining-up, while somewhat pretty, is not how we usually format code in Grapht and LensKit. We just use one space to separate each of the elements.